### ISSUE TITLE
List of "underfilled pages" only returns list of pages when in 2-col mode

### ISSUE THREAD

**Issue by markpenny** [sillsdev/ptx2pdf#1006]
It doesn't seem to show me a list when I'm in Single-col mode, but this would also be useful.

![Image](https://github.com/user-attachments/assets/221cbb14-6635-49a9-a609-c9b43dd5ddc3)

**Comment by davidc86** [comment]
Ah, okay this relates to my recent issue #1010 
In our area, we exclusively use the single-column formatting. So getting this to work on single-column pages would be a huge boost for us.

**Comment by markpenny** [comment]
Yes, this is a priority; and we might be able to squeeze it in before the end of the year. If not, then hopefully early next year.

**Comment by davidc86** [comment]
It looks like you did! Here is what I'm seeing now:

![Image](https://github.com/user-attachments/assets/37f0e2d0-bf3a-4a12-86a9-23b13d57ec5f)

So, I'm satisfied if you are!
